### PR TITLE
Add support for abbr tags on mobile

### PR DESCRIPTION
**Pull request by rlh1994**
Adds support to the abbreviations to work on mobile as currently they do nothing.

We both know I didn't work this out myself https://bitsofco.de/making-abbr-work-for-touchscreen-keyboard-mouse/ + https://stackoverflow.com/questions/34276581/display-abbr-and-acronym-on-mobile-devices

I can't make it delay the pop-up like OG abbr did...

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -80,6 +80,7 @@
   --ifm-color-primary-lightest: #b198df;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
   --ifm-font-color-base: rgb(218, 221, 225);
+  --ifm-font-color-base-inverse: #000000
```
Comment (stanch) [inline src/css/custom.css:83]:
```suggestion
  --ifm-font-color-base-inverse: black;
```

File: src/css/custom.css [hunk 2]
```diff
@@ -319,3 +320,30 @@ img.shield {
   border-radius: var(--ifm-alert-border-radius);
   transition: 0.4s cubic-bezier(.38,.14,.58,1) all;
 }
+
+abbr[data-title] {
+  position: relative;
+
```
Comment (stanch) [inline src/css/custom.css:326]:
```suggestion
cursor: help;
```

File: src/css/custom.css [hunk 3]
```diff
@@ -319,3 +320,30 @@ img.shield {
...
+  bottom: -30px;
+  width: auto;
+  white-space: nowrap;
+
+  /* style tooltip */
+  background-color: var(--ifm-font-color-base);
+  color: var(--ifm-font-color-base-inverse);
+  border-radius: 3px;
+  box-shadow: 1px 1px 5px 0 rgba(0, 0, 0, 0.4);
+  font-size: 14px;
```
Comment (stanch) [inline src/css/custom.css:347]:
```suggestion
  font-size: 14px;
  font-weight: normal;
```

**Review by stanch — CHANGES_REQUESTED** [review]

File: src/css/custom.css (hunk 1 above)
Comment (rlh1994) [reply src/css/custom.css:83]: I just copied the one from the font colour base in the light theme?

Comment (stanch) [reply src/css/custom.css:83]: Fine by me, but add the missing `;` :)

**Review by stanch — APPROVED** [review]

**Pull request merged**